File: mobile/src/main/java/com/siliconlabs/bledemo/Browser/Model/Logs/DisconnectByButtonLog.java
package com.siliconlabs.bledemo.Browser.Model.Logs;

import com.siliconlabs.bledemo.Browser.Model.LogType;

public class DisconnectByButtonLog extends Log {

    public DisconnectByButtonLog(String deviceAddress) {
        setLogTime(getTime());
        setLogInfo(deviceAddress + " Disconnected on UI");
        setLogType(LogType.INFO); //malo wazne
        setDeviceAddress(deviceAddress);
    }

}

File: mobile/src/main/java/com/siliconlabs/bledemo/Browser/Models/Logs/DisconnectByButtonLog.kt
package com.siliconlabs.bledemo.Browser.Models.Logs

import com.siliconlabs.bledemo.Browser.Models.LogType

class DisconnectByButtonLog(deviceAddress: String) : Log() {
    init {
        logTime = getTime()
        logInfo = "$deviceAddress Disconnected on UI"
        logType = LogType.INFO
        super.deviceAddress = deviceAddress
    }
}
